src/browser/screen.js

```diff
@@ -67,86 +101,259 @@ function ScreenAdapter(options, screen_fill_buffer)
         return "#" + "0".repeat(6 - n.length) + n;
     }
 
+    function render_font_bitmap(src_bitmap)
+    {
+        const dst_size = 8 * 256 * font_width * font_height;
+        const dst_bitmap = font_bitmap && font_bitmap.length === dst_size ?
+            font_bitmap : new Uint8ClampedArray(dst_size);
```

Comment:
This inspects the global `font_bitmap`, returns it and then the caller overwrites the same global, which is a bit tedious. How about:

```js
if(font_bitmap.length !== dst_size) font_bitmap = new Uint8ClampedArray(dst_size)
// use font_bitmap in the rest of this function
```

If it doesn't cause problems with the rest of the code, we could also change the condition to `font_bitmap.length < dst_size`, so repeated font resizes don't cause allocations.

Reply:
Sure, both resolved in 403d5e5.